app/controllers/application_controller.rb

```diff
@@ -19,9 +19,9 @@ class ApplicationController < ActionController::Base
     render "errors/forbidden", status: :forbidden, formats: :html
   end
 
-  def check_service_unavailable
-    if Settings.service_unavailable
-      render "errors/service_unavailable", status: :service_unavailable, formats: :html
+  def check_maintenance_mode_is_enabled
+    if Settings.maintenance_mode_enabled
+      redirect_to maintenance_page_path
```

Comment:
Do we still need to get rid of the `:service unavailable` status if we're exempting the ping action from running `check_maintenance_mode_is_enabled`?

Reply:
We happy to keep it as :ok/200 status code because we are aware of why users are getting the page. its also redirected to `/maintenance` so we can track number of times users are hitting this page.